common.blocks/form-field/_type/form-field_type_hidden.browser.js

```diff
@@ -0,0 +1,18 @@
+modules.define('form-field', function(provide, FormField) {
+
+    provide(FormField.decl({ modName : 'type', modVal : 'hidden' }, {
+
+        getName : function() {
+            return this.params.name;
+        },
+
+        setVal : function(val) {
+            this.params.value = val;
```

Comment:
It's definitely a confusing thing because user will expect these values on the server side.